app/src/lib/source-map-support.ts

```diff
@@ -32,7 +32,7 @@ function retrieveSourceMap(source: string) {
 
   if (__DEV__ && path.startsWith('http://')) {
     try {
-      const xhr = new XMLHttpRequest()
+      const xhr = new window.XMLHttpRequest()
```

Comment:
I'm not quite sure this is the best approach, as these changes now cause the TS compiler to consider these errors. I'm also not across how best to mash tsc and eslint together at this point, so I'm not sure how to proceed...